xml/System.Numerics/Complex.xml

```diff
@@ -1858,11 +1898,26 @@
         <Parameter Name="right" Type="System.Numerics.Complex" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="left">The double-precision real value to add.</param>
+        <param name="right">The complex value to add.</param>
+        <summary>Adds a double-precision real number to a complex number.</summary>
+        <returns>The sum of <paramref name="left" /> and <paramref name="right" /> as a complex number.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The addition of a real number (which can be regarded as the complex number a + 0i) and a complex number (c + di) takes the following form:
```

Comment:
Why did you put this paragraphs back? I think is redundant with the `MemberGroup`'s Remarks section.

Reply:
Tanner requested to have a specific remark for each overload, similar to how it was originally. I added a general remark with shared information, and the specific remarks have information relevant to the particular overload.

Reply:
I think what he meant was to add a note stating that the overloads taking a double are significant more optimal to the one that takes two complex numbers.
As stated in the issue https://github.com/dotnet/corefx/issues/15927, multiplying a complex by a double takes 2 flops while multiplying two complexes takes 6 flops.

Reply:
Right, I wanted to have the general remarks still for everything, but a comment indicating that this overload was more efficient if possible.

If we think they can be worded generally, add it to the general remarks. Otherwise, just to the double overload remarks.